### ISSUE TITLE
Add expire capability for SET calls to Redis

### ISSUE THREAD

**Issue by uzunbaz** [jcustenborder/kafka-connect-redis#26]
Currently, there is no capability to configure an expiration for records pushed to Redis by this connector. It will be nice to have a config to set the expire time for the records.

In other words, instead of `SET key value`, it should call `SET key value  [EX seconds|PX milliseconds]` where the time duration is configurable with the json file passed to the connector.

**Comment by jcustenborder** [comment]
@uzunbaz What are you thinking? Just a configuration setting with the number of milliseconds until an expiration and then applying that to all `SET` calls?

**Comment by uzunbaz** [comment]
Yes, something like that. I checked the source code briefly and looks like `MSET` async command is used for setting the keys. However, Redis `MSET` command does not take any expiration ttl as a parameter (there are tickets/discussions that requires this, e.g. : https://github.com/antirez/redis/issues/5867)

So, we can either use `SET EX` instead of `MSET` or have another call for expiration after `MSET`, lettuce api has a function for expire: `AbstractRedisAsyncCommands.expire(...)` ([link to lettuce source](https://github.com/lettuce-io/lettuce-core/blob/f140277a223aa79d0e39428e457a620702fd0ec7/src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java#L524)).

**Comment by mbrembilla** [comment]
Hi, are there any news about this enhancement?
It would be really helpfull to have a way to set the expire time on the keys.

**Comment by vinaygopalkrishnan** [comment]
Hi, 
Is this enhancement implemented or planned? It would be very helpful to have this for cache maintenance.

**Comment by gcaragea** [comment]
Bumping this up, having this functionality would enable our use-case for caching. Much needed please!

**Comment by andlien** [comment]
Would also love to have this implemented. Perhaps even with the possibility to set it dynamically with a kafka-header? Which could be added to the sink-configuration?

**Comment by jcustenborder** [comment]
Ignoring the fact that we need to use `SET` instead of `MSET` how would we want to calculate the the value for `EX`? As a static config setting? So something like `EX 60` with 60 being pulled from config. Another alternative would be `EXAT 19871341` calculated based on the timestamp?

**Comment by varunpathania** [comment]
Hi, was this enhancement implemented?
